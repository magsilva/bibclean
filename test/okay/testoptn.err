%% testoptn.bib:11:Invalid checksum for ISSN 0302-9744 in ``ISSN = "0302-9744 (bad check digit), 1611-334X (bad check digit)"''.
%% testoptn.bib:11:Invalid checksum for ISSN 1611-334X in ``ISSN = "0302-9744 (bad check digit), 1611-334X (bad check digit)"''.
%% testoptn.bib:13:Invalid checksum for ISSN 0302-9742 in ``ISSN-L = "0302-9742 (bad check digit)"''.
%% testoptn.bib:13:Unexpected ISSN-L field length in ``"0302-9742 (bad check digit)"''.
%% testoptn.bib:15:Invalid checksum for ISSN 0302-9742 in ``ISSN-L = "03029742"''.
%% testoptn.bib:15:Unexpected ISSN-L field length in ``"03029742"''.
