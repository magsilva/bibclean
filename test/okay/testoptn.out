@Article{Barker:2012:VPW,
  author =       "Ken Barker",
  title =        "``Valuing'' Privacy While Exposing Data Utility",
  journal =      j-LECT-NOTES-COMP-SCI,
  volume =       "6121",
  pages =        "1--2",
  year =         "2012",
  CODEN =        "LNCSD9",
  DOI =          "http://dx.doi.org/10.1007/978-3-642-25704-9_1",
  ISSN =         "0302-9743 (print), 1611-3349 (electronic)",
  ISSN =         "0302-9744 (bad check digit), 1611-334X (bad check
                 digit)",
  ISSN-L =       "0302-9743",
  ISSN-L =       "0302-9742 (bad check digit)",
%%  bad ISSN-L length in next line
    ISSN-L =     "03029742",
  bibdate =      "Wed Dec 19 15:20:12 MST 2012",
  bibsource =    "http://www.math.utah.edu/pub/tex/bib/lncs2012a.bib",
  URL =          "http://link.springer.com/accesspage/chapter/10.1007/978-3-642-25704-9_1",
  acknowledgement = ack-nhfb,
  book-DOI =     "http://dx.doi.org/10.1007/978-3-642-25704-9",
  book-URL =     "http://www.springerlink.com/content/978-3-642-25704-9",
  fjournal =     "Lecture Notes in Computer Science",
}
